.github/workflows/build-on-pull-request.yml

```
@@ -20,13 +20,17 @@ jobs:
     permissions:
       contents: read
 
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
```

actionlint (1.7.7)

property "deployment" is not defined in object type {}

(expression)